src/plugins/intel_gpu/src/graph/batch_to_space.cpp

```diff
@@ -66,6 +68,94 @@ layout batch_to_space_inst::calc_output_layout(batch_to_space_node const& node,
     return layout{output_type, input_format, desc->out_size};
 }
 
+static std::vector<int32_t> tensorToVec(const tensor& t, const format f) {
```

Comment:
nit: could you replace function name with snake case using underscore instead of camel case to align with coding convention of graph level code in gpu plugin?

Reply:
done